### DISCUSSION TITLE
Getting event merging to work

### DISCUSSION THREAD

**Discussion by jessoliva** [coedit/gcal-rainbow-surf#6] [Q&A] [answered]
Hi! I haven't been able to get this to work, not 100% sure why. 
I would really appreciate any help because I would love to use this!

Thanks!
Jess

**Comment by ezdub** [comment]
Hey there- Thanks for reaching out, I'd love to help you get this to work.

This extension is helpful when you have the same event on two or more of your calendars & it does its magic automatically without you pressing any buttons.
For example, if a coworker or family member has shared their calendar with you so you see all their events and you're both invited to the same event, Google Calendar would normally show that there are two events at the same time, one block in each calendar's color.
However, with this extension installed, those two event blocks automatically become one. 

See this image for an example where my blue and orange calendars were both invited to the "Adrian — Z" event so the extension makes calendar.google.com show one block composed of both colors.
Please send a screen shot of what it looks like on your end if you're not getting the same results.

<img width="476" alt="Screenshot of Rainbow.surf in action" src="https://github.com/coedit/gcal-rainbow-surf/assets/1911998/fd39357f-f8cb-46e5-a32a-22adb5db0879">

I hope this is helpful, please let me know.
-Z

**Reply by jessoliva** [reply]
hi Z! Thank you for your response :)
That looks great! 
In my case, I have my calendar with an event and I was sent an invite to the same event that I accepted, it looks like this:
<img width="306" alt="Screenshot 2023-06-20 at 1 49 11 PM" src="https://github.com/coedit/gcal-rainbow-surf/assets/95152563/420e98d3-2115-4f70-8022-7c0d45bb4de2">

Would the extension work for this type of situation?

Thank you,
Jess

**Reply by ezdub** [reply, marked as answer]
Hi Jess-
I apologize for the delayed response. 

I created the same event on the same calendar and the extension merged it into one event in the same color as the calendar.
Note that the title and everything displayed in the box must match exactly. Occasionally, the page must be reloaded for the merge to show after an event is added or updated.


Please let me know if this is not helpful
-Z

**Reply by jessoliva** [reply]
hi Z! 

Thanks so much for explaining that. The dash for the events were different, one was slightly longer! Fixed that and it merged! yay :) 
thank you for helping me fix that!

best,
jess
